### PR TITLE
Fix/issues

### PR DESCRIPTION
**Pull request by Arkant**

### CODE REVIEW

File: src/app/index.module.js [hunk 1]
```diff
@@ -25,9 +26,11 @@ const App = angular.module(
   "angularApp", [
     // plugins
     uiRouter,
+    toastr,
```
Comment (PodstDO) [inline src/app/index.module.js]: Useless because You include it on line 33

**Review by PodstDO — CHANGES_REQUESTED** [review]

File: src/app/index.module.js (hunk 1 above)
Comment (Arkant) [reply src/app/index.module.js]: Visa versa, 29 string is right 33 wrong. Fixed it.

**Review by PodstDO — DISMISSED** [review]

File: src/app/index.module.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import config from './index.config';
 import run from './index.run';
 
 import uiRouter from '@uirouter/angularjs';
-
+import '../../node_modules/angular-toastr/dist/angular-toastr.min.css';
```
Comment (motifej) [inline src/app/index.module.js]: it is not place for it!

File: src/app/index.module.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ import adminModule from './pages/admin/admin.module';
 import converterModule from './pages/currency-converter/converter.module';
 import homeModule from './pages/home/home.module';
 import signUpModule from './pages/sign-up/sign-up.module';
+import toastr from 'angular-toastr';
```
Comment (motifej) [inline src/app/index.module.js]: it is not place for it!

File: src/app/pages/sign-in/sign-in.controller.js [hunk 4]
```diff
@@ -1,18 +1,25 @@
...
+class SignInController {
+    constructor($scope, authenticationService, toastr) {
+        'ngInject';
+        this.scope = $scope;
+        this.authenticationService = authenticationService;
+        this.toastr = toastr;
+      }
+      
+      $onInit () {
+        this.scope.signIn = () => {
```
Comment (motifej) [inline src/app/pages/sign-in/sign-in.controller.js]: it is not correct if you use class( put all function to this)

File: src/app/pages/sign-in/sign-in.controller.js [hunk 5]
```diff
@@ -1,18 +1,25 @@
...
+        const { email, password } = this.scope.user;
+        
+        this.authenticationService.signInToFirebase(email, password)
+        .then(response => {
+          if (response) {
+            this.toastr.error(`${ response.message }`);
```
Comment (motifej) [inline src/app/pages/sign-in/sign-in.controller.js]: why do you use template string?

File: src/app/pages/sign-in/sign-in.module.js [hunk 6]
```diff
@@ -3,7 +3,7 @@
 import SignInController from './sign-in.controller.js';
 import './sign-in.scss';
 
-const signInModule = angular.module('signInModule', []);
+const signInModule = angular.module('signInModule', ['toastr']);
```
Comment (motifej) [inline src/app/pages/sign-in/sign-in.module.js]: toastr - dependency of global app, not for module

**Review by motifej — CHANGES_REQUESTED** [review]

File: src/app/index.module.js (hunk 2 above)
Comment (Arkant) [reply src/app/index.module.js]: Removed it

File: src/app/pages/sign-in/sign-in.controller.js (hunk 4 above)
Comment (Arkant) [reply src/app/pages/sign-in/sign-in.controller.js]: Done

File: src/app/pages/sign-in/sign-in.controller.js (hunk 5 above)
Comment (Arkant) [reply src/app/pages/sign-in/sign-in.controller.js]: My bad.

File: src/app/pages/sign-in/sign-in.module.js (hunk 6 above)
Comment (Arkant) [reply src/app/pages/sign-in/sign-in.module.js]: Ok, solved

**Review by Vladyslav-Shvorobuk — APPROVED** [review]

**Review by Alexandra1003 — APPROVED** [review]

**Review by PodstDO — APPROVED** [review]

**Pull request merged**